fontforgeexe/charview.c

```diff
@@ -900,34 +900,13 @@ static void DrawPoint( CharView *cv, GWindow pixmap, SplinePoint *sp,
     char buf[16];
     int isfake;
 
-    if ( DrawOpenPathsWithHighlight
```

Comment:
So this is basically removing this option (`DrawOpenPathsWithHighlight`)?

Reply:
No, it's still there for the line. This just stops painting the points differently when the contour is open.

Reply:
Ah right.